app/models/cash_transfer.rb

```diff
@@ -86,7 +86,8 @@ class CashTransfer < Ekylibre::Record::Base
   validate do
     errors.add(:reception_cash_id, :invalid) if reception_cash_id == emission_cash_id
     if transfered_at
-      errors.add(:transfered_at, :not_opened_financial_year) unless opened_financial_year?
+      errors.add(:transfered_at, :not_opened_financial_year) unless opened_financial_year? || financial_year_in_closure_preparation?
```

Comment:
Can you wrap this in a `if(closure_preparation?)/else` instead of `unless ... closure_preparation? + if closure_preparation?` ?